modules/configuring-vsphere-regions-zones.adoc

```diff
@@ -0,0 +1,160 @@
+|`datacenter` | IBMCloud
+|`computeCluster` | /IBMCloud/host/vcs-mdcnc-workload-1
+|`datastore` | /IBMCloud/datastore/workload_share_vcsmdncworkload_Yfyf6
+|`resourcePool`|/IBMCloud/host/cluster1/Resources/ResourcePool
+|`folder`|/IBMCloud/vm/test_folder
+|===
+
+.Prerequisites
+* You have an existing `install-config.yaml` installation configuration file.
+
+.Procedure
+
+. Use the `govc` command-line tool to create two vCenter tag categories: 
++
+[source,terminal]
+----
+$ govc tags.category.create -d "<region_tag_category_name>" <region_tag_category>
+
+$ govc tags.category.create -d "<zone_tag_category_name>" <zone_tag_category>
+----
+
+. Create region tags:
++
+[source,terminal]
+----
+$ govc tags.create -c <region_tag_category> <region_tag_1>
+
+$ govc tags.create -c <region_tag_category> <region_tag_2>
+----
+
+. Create four zone tags.
++
+[source,terminal]
+----
+$ govc tags.create -c <zone_tag_category> <zone_tag_1>
+
+$ govc tags.create -c <zone_tag_category> <zone_tag_2>
+
+$ govc tags.create -c <zone_tag_category> <zone_tag_3>
+
+$ govc tags.create -c <zone_tag_category> <zone_tag_4>
+----
+
+. Attach region tags to vCenter datacenter objects:
++
+[source,terminal]
+----
+$ govc tags.attach -c <region_tag_category> <region_tag_1> /<datacenter_1>
+
+$ govc tags.attach -c <region_tag_category> <region_tag_2> /<datacenter_2>
+----
+
+. Attach the zone tags to the vCenter datacenter objects:
++
+[source,terminal]
+----
+$ govc tags.attach -c <zone_tag_category> <zone_tag_1> /<datacenter_1/host/vcs-mdcnc-workload-1
+
+$ govc tags.attach -c <zone_tag_category> <zone_tag_2> /<datacenter_1/host/vcs-mdcnc-workload-2
+
+$ govc tags.attach -c <zone_tag_category> <zone_tag_3> /<datacenter_1/host/vcs-mdcnc-workload-3
+
+$ govc tags.attach -c <zone_tag_category> <zone_tag_4> /<datacenter_2/host/vcs-mdcnc-workload-4
+----
+
+. Create the `install-config.yaml` file and configure the file to support operating multiple datacenters in a vSphere center.
++
+[IMPORTANT]
+====
+You must specify at least one failure domain for your {product-title} cluster, so that you can provision datacenter objects for your VMware vCenter server. Consider specifying multiple failure domains if you need to provision virtual machine nodes in different datacenters, clusters, datastores, and so on.
+====
+
+. Change to the directory that contains the installation program and initialize the cluster deployment according to your chosen installation requirements.
+
+.Sample `install-config.yaml` file with multiple datacenters defined in a vSphere center
+
+[source,yaml]
+----
+...
+compute:
+...
+  vsphere:
+      zones:
+        - "<zones_tag_1>"
+        - "<zones_tag_2>"
+...
+controlPlane:
+...
+vsphere:
+      zones:
+        - "<zones_tag_1>"
+        - "<zones_tag_2>"
+...
+  vsphere:
+    vcenters:
+...
+    datacenters:
+      - <datacenter1_name>
+      - <datacenter2_name>
+    failureDomains:
+    - name: <zones_tag_1>
```

Comment:
FailureDomain.Name is arbitrary _but_ if used in machinepool (control plane, compute or default) **must** match a listed entry in `zones`.

Reply:
https://github.com/openshift/installer/blob/master/pkg/asset/machines/vsphere/machinesets.go#L102-L106